soc/nxp/imxrt/rt10xx/Kconfig.soc

```diff
@@ -0,0 +1,238 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMXRT_10XX
+	bool
+	select SOC_FAMILY_IMXRT
+
+config SOC_SERIES
+	default "rt10xx" if SOC_SERIES_IMXRT_10XX
```

Comment:
```suggestion
	default "imxrt_10xx" if SOC_SERIES_IMXRT_10XX
```
Name should match value here and in soc.yml

Reply:
it does match?

Reply:
it does not